internal/notifier/util.go

```diff
@@ -59,10 +59,29 @@ func formatNameAndDescription(event eventv1.Event) (string, string) {
 // involved object kind and name.
 func generateCommitStatusID(providerUID string, event eventv1.Event) string {
 	uidParts := strings.Split(providerUID, "-")
-	id := fmt.Sprintf("%s/%s/%s", event.InvolvedObject.Kind, event.InvolvedObject.Name, uidParts[0])
+	metadataLabelsSuffix := createMetadataLabelsSuffix(event)
```

Comment:
This has the potential to result in a very long string which is undesirable. We need to think on a way to constrain this to a certain length.

Reply:
true, it might be an option to change eventMetadata keys accepted for building an extended commit status ID? @makkes

Reply:
any suggestion how to proceed @makkes @stefanprodan ?